.github/workflows/tests.yaml

```diff
@@ -18,8 +18,8 @@ jobs:
       - name: Run unit tests
         run: |
           go test -v ./...
-  integration-test:
-    name: "Integration tests (go v${{ matrix.go-version }})"
+  cinder-integration-test:
```

Comment:
Please revert the name back to `integration-test` and merge contents of `manila-integration-test` here.

Reply:
done

Reply:
Thank you